scipy/stats/stats.py

```diff
@@ -3999,6 +3996,51 @@ def pearsonr(x, y):
     >>> stats.pearsonr([1, 2, 3, 4, 5], [10, 9, 2.5, 6, 4])
     (-0.7426106572325057, 0.1505558088534455)
 
+    It is important to keep in mind that no correlation does not imply
+    independence unless (x, y) is Gaussian. Correlation can even be zero
```

Comment:
Does "(x, y) is Gaussian" mean "sample pairs are independent and identically distributed and follow a bivariate normal distribution"?